envs/tensorflow-serving-env.yaml

```diff
@@ -0,0 +1,14 @@
+imported_envs:
```

Comment:
Should we have {% protobuf == "3.9.2" %} here as this environment file should not get executed for newer protobuf?